code/io/timer.h

```diff
@@ -157,6 +157,8 @@ int timestamp(int delta_ms);
 TIMESTAMP _timestamp(int delta_ms);		// use a leading underscore for now until all timestamps are converted over
 UI_TIMESTAMP ui_timestamp(int delta_ms);
 
+float missiontime_get_float();
```

Comment:
Ah, yeah. This is to do with a different set of curves I decided to not include in this PR because of how this PR is huge. Must have missed this when I was stripping the various parts out.